### ISSUE TITLE
Doc of integration between bitbucket source connector and airbyte not found

### ISSUE THREAD

**Issue by luancaarvalho** [faros-ai/airbyte-connectors#658]
Hello !
The docs about how to use bitbucket source connector within Airbyte is reaching “page not found “
Could you tell me how to integrate both or fix the docs ?

**Comment by tovbinm** [comment]
Yeah, we need to open a PR with Airbyte code repo to have it documented. @cjwooo is this the process?
